ui/k-toolbar/main.kytos

```diff
@@ -1,343 +1,431 @@
-      $.ajax({
-        async: true,
-        dataType: "json",
-        url: this.$kytos_server_api + "kytos/topology/v3",
-
-        success: function(data) {
-          self.switches = data['topology']['switches']
-          self.links = data['topology']['links']
-        }
-      });
-
-    },
-    show() {
-       var content = {
-                      "component": 'kytos-pathfinder-k-info-panel-best_path',
-                      "content": this.paths,
-                      "icon": "compass",
-                      "title": "Best Paths",
-                      "subtitle": "by kytos/pathfinder"
-                     }
-       this.$kytos.$emit("showInfoPanel", content)
-
-   }
-  },
-
-  computed: {
-    metric_options(){
-      var metric_options = {}
-
-      var bandwidth_options = []
-      var reliability_options = []
-      var delay_options = []
-      var utilization_options = []
-      var priority_options = []
-      var ownership_options = []
-      var spf_attribute_options = []
-
-      bandwidth_options.push({value: false, description: 'Bandwidth', selected: true});
-      bandwidth_options.push({value: true, description: 'Bandwidth (Flexible)'});
-      
-      reliability_options.push({value: false, description: 'Reliability', selected: true});
-      reliability_options.push({value: true, description: 'Reliability (Flexible)'});
-
-      delay_options.push({value: false, description: 'Delay', selected: true});
-      delay_options.push({value: true, description: 'Delay (Flexible)'});
-
-      utilization_options.push({value: false, description: 'Utilization', selected: true});
-      utilization_options.push({value: true, description: 'Utilization (Flexible)'});
-
-      priority_options.push({value: false, description: 'Priority', selected: true});
-      priority_options.push({value: true, description: 'Priority (Flexible)'});
-
-      ownership_options.push({value: false, description: 'Ownership', selected: true});
-      ownership_options.push({value: true, description: 'Ownership (Flexible)'});
-
-      metric_options["bandwidth"] = bandwidth_options
-      metric_options["reliability"] = reliability_options
-      metric_options["delay"] = delay_options
-      metric_options["utilization"] = utilization_options
-      metric_options["priority"] = priority_options
-      metric_options["ownership"] = ownership_options
-
-      return metric_options;
+        });
+      },
+      fetch_dpids: function() {
+        var self = this // create a closure to access component in the callback below
+          dataUrl = "/api/kytos/topology/v3/interfaces"
+          // Autocomplete usage example.
+          fetch(dataUrl).then(response => response.json())
+                      .then(data => {
+                          dpids = []
+                          for ( const [key,value] of Object.entries( data.interfaces ) ) {
+                              let item = key;
+                              if(value.name) {
```

Comment:
It should only set the item as `value.name+ " - " + item;` if `metadata.port_name` isn't set, otherwise `metadata.port_name` is supposed to be used where `value.name` is being used as mentioned in the issue requirements, could you refactor and adapt this accordingly?

Reply:
![20230525_175804](https://github.com/kytos-ng/pathfinder/assets/1010796/40a93e38-b7b1-40bb-a9c6-80895d6524f7)

For instance, here's `port_name` as `"intf1"` on an interface_id `00:00:00:00:00:00:00:01:1`, in this case, it shouldn't be using `s1-eth1` (its name)